test-gui.sh

```diff
@@ -5,10 +5,14 @@ set -exuo pipefail
 cd frontend
 
 yarn install
-if [ -z "${CHROME_VERSION-}" ]; then
+if [ $BRIDGE_E2E_BROWSER_NAME == 'firefox' ]; then
```

Comment:
This needs to be quoted. I'd also expect this to fail if the env var is not set because of `set -u`. So maybe
```suggestion
if [ "${BRIDGE_E2E_BROWSER_NAME-} == 'firefox' ]; then
```

Reply:
I see, thanks